2016-04-08.md

```diff
@@ -0,0 +1,37 @@
+We are thrilled to announce that Tessel 2 has shipped to all domestic and international pre-orders.
+
+To our knowledge, this is the first time a distributed, unpaid team of open source contributors has shipped a physical product. This is a major milestone in the open source hardware movement.
+
+Huge thanks to everyone who has contributed so far to the Tessel Project. Whether you're a core contributor, someone who found a typo to fix, or one of our test subjects for the start experience: thank you. We value what you've done.
+
+### "This sounds like a cool project, how can I get involved?"
```

Comment:
We should mention the https://github.com/tessel/onboarding repo here.

We should also suggest they check out https://github.com/tessel/t2-cli/issues?q=is%3Aopen+is%3Aissue+label%3Acontribution-easy